<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgLights</class>
 <widget class="QDialog" name="CQDlgLights">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Properties</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>411</width>
     <height>151</height>
    </rect>
   </property>
   <property name="title">
    <string>Main properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QCheckBox" name="qqEnabled">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Light is enabled</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="qqLocal">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>25</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Light is local</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>65</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Spot exponent</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqSpotExponent">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>65</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqConstantFactor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>85</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>85</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Constant attenuation factor</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>105</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Linear attenuation factor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqLinearFactor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>105</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>125</y>
      <width>221</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Quadratic attenuation factor</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqQuadraticFactor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>125</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqSpotCutoff">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>45</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>45</y>
      <width>201</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Spot cutoff [deg]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>411</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>Visual properties</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QToolButton" name="qqLightColor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>25</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust light color</string>
    </property>
   </widget>
   <widget class="QToolButton" name="qqCasingColor">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>45</y>
      <width>171</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Adjust casing color</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="qqSize">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>25</y>
      <width>76</width>
      <height>18</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>25</y>
      <width>131</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>Object size [m]</string>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqEnabled</tabstop>
  <tabstop>qqLocal</tabstop>
  <tabstop>qqSpotCutoff</tabstop>
  <tabstop>qqSpotExponent</tabstop>
  <tabstop>qqConstantFactor</tabstop>
  <tabstop>qqLinearFactor</tabstop>
  <tabstop>qqQuadraticFactor</tabstop>
  <tabstop>qqSize</tabstop>
  <tabstop>qqLightColor</tabstop>
  <tabstop>qqCasingColor</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
